test/e2e/dra/dra.go

```diff
@@ -2447,6 +2447,133 @@ var _ = framework.SIGDescribe("node")(framework.WithLabel("DRA"), func() {
 			}
 			drautils.TestContainerEnv(ctx, f, pod, pod.Spec.Containers[0].Name, false, containerEnv...)
 		})
+
+		ginkgo.It("must prevent overcommitment of extended resources", func(ctx context.Context) {
+			// Create first pod consuming all available resources
+			pod1 := b.Pod()
+			res := v1.ResourceList{
+				v1.ResourceName(b.ExtendedResourceName(0)): resource.MustParse("10"), // All available
+			}
+			pod1.Spec.Containers[0].Resources.Requests = res
+			pod1.Spec.Containers[0].Resources.Limits = res
+			b.Create(ctx, pod1)
+			err := e2epod.WaitForPodRunningInNamespace(ctx, f.ClientSet, pod1)
+			framework.ExpectNoError(err, "start pod")
+
+			ginkgo.By("Check that pod is processed by the DRA driver")
+			containerEnv := []string{
+				"container_0_request_0", "true",
+			}
+			drautils.TestContainerEnv(ctx, f, pod1, pod1.Spec.Containers[0].Name, false, containerEnv...)
+
+			// Second pod should remain unschedulable
+			pod2 := b.Pod()
+			pod2.Spec.Containers[0].Resources.Requests = res
+			pod2.Spec.Containers[0].Resources.Limits = res
+			b.Create(ctx, pod2)
+
+			framework.ExpectNoError(e2epod.WaitForPodNameUnschedulableInNamespace(ctx, f.ClientSet, pod2.Name, f.Namespace.Name))
+
+			// After deleting pod1, pod2 should be schedulable and run successfully
+			b.DeletePodAndWaitForNotFound(ctx, pod1)
+			err = e2epod.WaitForPodRunningInNamespace(ctx, f.ClientSet, pod2)
+			framework.ExpectNoError(err, "start pod")
+			drautils.TestContainerEnv(ctx, f, pod2, pod2.Spec.Containers[0].Name, false, containerEnv...)
+		})
+
+		ginkgo.It("must reject pod with invalid extended resource name", func(ctx context.Context) {
```

Comment:
this case is for general extended resource, it is not specific to DRA extended resource. so it may not be necessary to add it here.